cmd/acikubectl/cmd/debug.go

```diff
@@ -393,14 +447,19 @@ func clusterReport(cmd *cobra.Command, args []string) {
 			hasErrors = true
 			continue
 		}
-
-		tarWriter.WriteHeader(&tar.Header{
-			Name:    cmd.name,
-			Mode:    0644,
-			ModTime: now,
-			Size:    int64(buffer.Len()),
-		})
-		buffer.WriteTo(tarWriter)
+		//To avoid double hostfiles
+		//if strings.Contains(cmd.name, "hostfiles") {
+		if cmd.skipOutputFile == true {
+			fmt.Printf("Skiping adding hostfile: %v to output: %v\n", cmd.name, output)
```

Comment:
This seems like a debug log, and I do realize that we are not putting things in a log file, but do we really need to print this? It kind of makes the entire log difficult to read (and probably confusing).